docs/docs/2-guides/3-supported-apis.mdx

```diff
@@ -9,46 +9,71 @@ import {ConsulAPIBadge} from "@site/src/components/CustomBadge";
+|                  | List Checks for Service           | GET /v1/health/checks/:id                      |   ✅    |
+|                  | List Nodes for Service            | GET /v1/health/service/:id                     |   ✅    |
+|                  | List Checks in State              | GET /v1/health/state/:state                    |   ✅    |
+| KV Store         | Read Key                          | GET /v1/kv/:key                                |   ✅    |
+|                  | Create/Update Key                 | PUT /v1/kv/:key                                |   ✅    |
+|                  | Delete Key                        | DELETE /v2/kv/:key                             |   ✅    |
+| Namespace        | List Namespaces                   | GET /v1/namespaces                             |   ✅    |
+|                  | Read Namespace                    | GET /v1/namespace/:name                        |   ✅    |
+|                  | Create Namespace                  | PUT /v1/namespace                              |   ✅    |
+|                  | Update Namespace                  | PUT /v1/namespace/:name                        |   ✅    |
+|                  | Delete Namespace                  | DELETE /v1/namespace/:name                     |   ✅    |
+| Session          | List Session                      | GET /v1/session/list                           |   ✅    |
+|                  | Create Session                    | POST /v1/session/create                        |   ✅    |
+|                  | Destroy Session                   | PUT /v1/session/destroy/:id                    |   ✅    |
+|                  | Renew Session                     | PUT /v1/session/renew/:id                      |   ✅    |
+|                  | Info Session                      | GET /v1/session/node/:node                     |   ✅    |
+|                  | Destroy Session                   | PUT /v1/session/destroy/:id                    |   ✅    |
+|                  | List Sessions for Node            | GET /v1/session/node/:node                     |   ✅    |
+| Status           | Status of Leader                  | GET /v1/status/leader                          |   ✅    |
+|                  | Status of Peers                   | GET /v1/status/peers                           |   ✅    |
+| Transaction      | Transaction                       | PUT /v1/txn                                    |   ✅    | 
+| Config           | Retrieve a specific configuration by kind and name | GET /v1/config/{kind}/{name}  |   ✅    |
+|                   | List all configuration entries of a specific kind | GET /v1/config/{kind}         |   ✅    |
+|                   | Set a new configuration entry                    | PUT  /v1/config                |   ✅    |
+|                   | Delete a specific configuration entry by kind and name | DELETE /v1/config/{kind}/{name}   |  ✅     |
```

Comment:
Not sure if it is going to render nicely.
Let's merge it in and see.

Reply:
It is ok 👍

Reply:
Cool 😎